media-libs/libjxl/libjxl0.11-0.11.0.recipe

```diff
@@ -76,43 +75,67 @@ CONFLICTS_devel="
 PROVIDES_tools="
 	libjxl0.11${secondaryArchSuffix}_tools = $portVersion
 	cmd:benchmark_xl$secondaryArchSuffix
+	cmd:cjpegli$secondaryArchSuffix
 	cmd:cjxl$secondaryArchSuffix
+	cmd:djpegli$secondaryArchSuffix
 	cmd:djxl$secondaryArchSuffix
 	cmd:jxlinfo$secondaryArchSuffix
 	"
 REQUIRES_tools="
-	$REQUIRES
 	libjxl0.11$secondaryArchSuffix == $portVersion base
-	lib:libstdc++$secondaryArchSuffix
+	$REQUIRES
+	lib:libavif$secondaryArchSuffix
+	lib:libgif$secondaryArchSuffix
+	lib:libIex_3_2$secondaryArchSuffix
+	lib:libIlmThread_3_2$secondaryArchSuffix
+	lib:libImath_3_1$secondaryArchSuffix
+	lib:libOpenEXR_3_2$secondaryArchSuffix
+	lib:libOpenEXRUtil_3_2$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+#	lib:libstdc++$secondaryArchSuffix
```

Comment:
Together with libatomic below it looks like these are not used/required, maybe they did before?